routes/users.js

```diff
@@ -52,10 +55,18 @@ router.post("/login", async (req, res, next) => {
     if (user) {
       const verifiedUser = await bcrypt.compare(password, user.password);
       if (verifiedUser) {
+        const token = md5(Date.now() + user.username);
+        const expiry = Date.now() + 1000 * 60 * 60;
+        const access_token = new AccessToken({
+          user_id: user._id,
+          access_token: token,
+          expiry,
+        });
+        await access_token.save();
```

Comment:
on every login new access will generate and save, don't do this
find existing token for user if available return that else create